packages/lib-classifier/src/store/SubjectViewerStore/Readme.md

```diff
@@ -0,0 +1,17 @@
+# SubjectViewerStore
+
+## Actions
+
+- **setOnPan:** register an onPan handler with the subject toolbar. Subject viewers should expose a method with the signature `onPan(dx, dy)` to enable panning.
+  - _dx_: -1 (left), 0 (ignored), 1 (right)
+  - _dy_: -1 (up), 0 (ignored), 1 (down)
+- **setOnZoom:** register an onZoom handler with the subject toolbar. Subject viewers should expose a method with the signature `onZoom(type, value)` to enable zooming.
+  - _type:_ `'zoomin'|'zoomout'|'zoomto'`
+  - _value:_ TBD
```

Comment:
Does anyone know what `value` controls in the `onZoom` handler?

Reply:
It's a number and it's the optional scale value for the zoom function, defaulting to 1.2 for zooming in and 0.8 for zooming out within the zoom code of both the TESS LCV and VXZoom. I think only the TESS LCV uses `zoomto` and I'm not exactly sure what it does. The VXZoom has a zoom to point function that is self contained in that class specifically for the double click event.